.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+    "presets": [
+      ["env", {
+        "targets": {
+          "node": "current"
```

Comment:
I know, it's nitpicking because it only matters when you actually use a language featuer.  :-)     

But I would target e.g. node 8.2.1+   it's very complete ( https://node.green/#ES2016 ) and we don't annoy people that don't update their node every two weeks.

Reply:
According to the documentation of Babel.`current` is using the node.js version where the babel script is executed. 

> For convenience, you can use "node": "current" to only include the necessary polyfills and transforms for the Node.js version that you use to run Babel:

Reply:
Ah, great!  RTFM....    learnt something useful now :-)